src/Okta.Sdk.IntegrationTests/PoliciesScenarios.cs

```diff
@@ -0,0 +1,1482 @@
+
+            try
+            {
+                createdPolicy.Should().NotBeNull();
+                createdPolicy.Name.Should().Be(policy.Name);
+                createdPolicy.Type.Should().Be(PolicyType.OktaSignOn);
+                createdPolicy.Status.Should().Be("ACTIVE");
+                createdPolicy.Description.Should().Be(policy.Description);
+                ((IOktaSignOnPolicy)createdPolicy).Conditions.Should().NotBeNull();
+                ((IOktaSignOnPolicy)createdPolicy).Conditions.People.Groups.Should().NotBeNull();
+                ((IOktaSignOnPolicy)createdPolicy).Conditions.People.Groups.Include.Should().HaveCount(1);
+                ((IOktaSignOnPolicy)createdPolicy).Conditions.People.Groups.Include.First().Should().Be(createdGroup.Id);
+            }
+            finally
+            {
+                await createdGroup.DeleteAsync();
+                await client.Policies.DeactivatePolicyAsync(createdPolicy.Id);
+                await client.Policies.DeletePolicyAsync(createdPolicy.Id);
+            }
+        }
+
+        [Fact]
+        public async Task GetAllPoliciesByType()
+        {
+            var client = TestClient.Create();
+
+            var policy1 = new OktaSignOnPolicy()
+            {
+                // Name has a maximum of 50 chars
+                Name = $"Sign On policy {Guid.NewGuid()}".Substring(0, 50),
+                Type = PolicyType.OktaSignOn,
+                Status = "ACTIVE",
+                Description = "The default policy applies in all situations if no other policy applies.",
+            };
+
+            var policy2 = new PasswordPolicy()
+            {
+                Name = $"Password policy {Guid.NewGuid()}".Substring(0, 50),
+                Type = PolicyType.Password,
+                Status = "ACTIVE",
+                Description = "The default policy applies in all situations if no other policy applies.",
+            };
+
+            var createdPolicy1 = await client.Policies.CreatePolicyAsync(policy1);
+            var createdPolicy2 = await client.Policies.CreatePolicyAsync(policy2);
+
+            try
+            {
+                var signOnPolicies = await client.Policies.ListPolicies(PolicyType.OktaSignOn).ToList();
+                signOnPolicies.Should().NotBeNullOrEmpty();
+                signOnPolicies.First(x => x.Id == createdPolicy1.Id).Should().NotBeNull();
+                signOnPolicies.FirstOrDefault(x => x.Id == createdPolicy2.Id).Should().BeNull();
+
+                var passwordPolicies = await client.Policies.ListPolicies(PolicyType.Password).ToList();
+                passwordPolicies.Should().NotBeNullOrEmpty();
+                passwordPolicies.First(x => x.Id == createdPolicy2.Id).Should().NotBeNull();
+                passwordPolicies.FirstOrDefault(x => x.Id == createdPolicy1.Id).Should().BeNull();
+            }
+            finally
+            {
+                await client.Policies.DeactivatePolicyAsync(createdPolicy1.Id);
```

Comment:
Use `await createdPolicy1.deactivate()` so that we also test those crud methods?  Do we have code coverage reporting on this lib?

Reply:
@robertjd I will add those cases too.